KeyCommands.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "KeyCommands"
+  s.version      = package['version']
+  s.summary      = "iOS UIKeyCommand native component for React Native"
+  s.license      = package['license']
+
+  s.authors      = "Fang-Pen Lin <hello@fangpenlin.com> (https://fangpenlin.com)"
```

Comment:
i would just use
```
s.authors      = package['author']
```
so that the Podspec reads all values from `package.json`

It seems strange that we use that format for some fields (version, license) but not others?

(see [react-native-webview](https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec) for an example)

Reply:
`package['author']` is a JSON object and it has to be a string in podspec. I wanted to minimize changes, but we could of course update package.json, too.